docs/host_for_multiuser/setup.md

```diff
@@ -62,90 +64,97 @@ Follow the steps below to setup `multiuser` mode.
-    - Set `MYSQL_USER` to {DB_USER_NAME}, which you created in the previous step.
-    - Set `MYSQL_PASSWORD` to {DB_USER_PASSWORD}, which you created in the previous step.
-    - `MYSQL_ROOT_PASSWORD` can be left commented.
-
-### Setup Frontend
-1. Install node.js(version 20)
-    - https://nodejs.org
-2. Install yarn
-    ```bash
-    npm install -g yarn
-    ```
-3. Install frontend requirements
-    ```bash
-    cd frontend
-    yarn install
-    ```
-4. Build frontend
-    ```bash
-    yarn build
-    ```
 
-### Setup Backend
-- See OptiNiSt installation guide.
+###### 2. Start backend (Database is set up on startup)
+```bash
+docker compose -f docker-compose.dev.multiuser.yml up studio-dev-be -d
+```
+
+###### 3. Insert initial data
+```bash
+docker exec -it {DB_DOCKER_CONTAINER_NAME} mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}
+```
+```sql
+INSERT INTO organization (name) VALUES ('{YOUR_ORGANIZATION_NAME}');
+INSERT INTO roles (id, role) VALUES (1, 'admin'), (20, 'operator');
+INSERT INTO users (uid, organization_id, name, email, active) VALUES ('{FIREBASE_UID}', 1, '{YOUR_NAME}', '{YOUR_EMAIL}', true);
+INSERT INTO user_roles (user_id, role_id) VALUES (1, 1);
+```
+- USER_UID is the user uid you created in the previous step ([Create admin user for the project](#create-admin-user-for-the-project)).
+- Only 2 roles, "admin" and "operator" are supported for now.
+  - "admin"
+    - can manage other users
+  - "operator"
+    - general user
+
+##### Run OptiNiSt
+```bash
+docker compose -f docker-compose.dev.multiuser.yml up -d
+```
+
+1. Access to `http://{YOUR_HOST}:8000` from your browser.
+2. Confirm that you can SingIn with your Firebase Authentication account.
+
+#### For Other Platforms
+
+##### Setup Backend
+- See [OptiNiSt installation guide](../installation/index.rst).
 - After create and activate conda environment for the project, run following commands
 
-1. Install backend requirements
-    ```bash
-    cd studio
-    pip install .
-    ```
-2. Setup database
-    ```bash
-    alembic upgrade head
-    ```
-3. Insert initial data
+###### 1. Set OptiNiSt config
+- Same as [[Set OptiNiSt config]](#set-optinist-config) procedure.
 
-    ```bash
-    mysql -u DB_USER_NAME -p
-    ```
-    ```sql
-    USE YOUR_DATABASE_NAME;
-    INSERT INTO organization (name) VALUES ('YOUR_ORGANIZATION_NAME');
-    INSERT INTO roles (id, role) VALUES (1, 'admin'), (20, 'operator');
-    INSERT INTO users (uid, organization_id, name, email, active, ) VALUES ('USER_UID', 1, 'YOUR_EMAIL', 'YOUR_PASSWORD', 1);
-    INSERT INTO user_roles (user_id, role_id) VALUES (1, 1);
-    ```
-    - USER_UID is the user uid you created in the previous step ([Create admin user for the project](#create-admin-user-for-the-project)).
-    - Only 2 roles, "admin" and "operator" are supported for now.
-      - "admin"
-        - can manage other users
-      - "operator"
-        - general user
-
-### Run OptiNiSt
+###### 2. Setup database
+```bash
```

Comment:
> Switch to the the optinist folder cloned in Clone the repository

That is correct, but since the explanation is a bit verbose, I will annotate OPTINIST_ROOT_PATH.